### PR TITLE
Allow Puppet 6 and remove stdlib dependency

### PR DESCRIPTION
**Pull request by alexjfisher**
By replacing `anchor` with `contain` (which has been available since
puppet 3.4), the dependency on puppetlabs/stdlib can be removed
altogether.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -17,18 +17,13 @@
...
-    anchor { 'augeas::begin': }
-    -> class {'::augeas::files': }
-    -> anchor { 'augeas::end': }
+    contain 'augeas::files'
   } else {
-    anchor { 'augeas::begin': }
-    -> class {'::augeas::packages': }
-    -> class {'::augeas::files': }
-    -> anchor { 'augeas::end': }
+    contain 'augeas::packages'
```
Comment (alexjfisher) [inline manifests/init.pp]: Perhaps all of this code (puppet < 4) can be removed in a separate PR.  I didn't want to stray too far off my original goal of making sure this module would be compatible with the imminent stdlib 6.0.0

Comment (raphink) [reply manifests/init.pp]: LGTM
